dbprocessing/tables.py

```diff
@@ -54,11 +54,11 @@ def definition(name):
             schema.Column('mission_id', types.Integer, autoincrement=True, primary_key=True,
                           nullable=False),
             schema.Column('mission_name', types.String(20), nullable=False, unique=True),
-            schema.Column('rootdir', types.String(50), nullable=False, ),
-            schema.Column('incoming_dir', types.String(50), nullable=False, ),
-            schema.Column('codedir', types.String(50), nullable=True, ),
-            schema.Column('inspectordir', types.String(50), nullable=True, ),
-            schema.Column('errordir', types.String(50), nullable=True, )
+            schema.Column('rootdir', types.String(250), nullable=False, ),
+            schema.Column('incoming_dir', types.String(250), nullable=False, ),
+            schema.Column('codedir', types.String(250), nullable=True, ),
+            schema.Column('inspectordir', types.String(250), nullable=True, ),
+            schema.Column('errordir', types.String(250), nullable=True, )
```

Comment:
Yes, our "incoming_dir" can be pretty far in the "/projects" directory (which is a network drive).  I changed it lately, but our directories have different baselines, satellite and sensors.   Maybe 250 is a bit big for each case, but I usually like long descriptive strings.  50 character is way to small.   (I just added a 2 before your 50...)

Not sure what you mean about adding it to the list of database changes?

Reply:
#7 has a list of changes we've made to the db over the time, so this should go in there (and we should add it to the migration script as well.) So longer names here will be part of a v1 database instead of our current mess of v0.